### ISSUE TITLE
Refined Storage disks disappear after reloading world

### ISSUE THREAD

**Issue by Pyeroh** [refinedmods/refinedstorage#3130]
#### What happens:
When I reload the world (in singleplayer, but I guess multiplayer has the same problem), my disks disappear, and all storage blocks become empty.

I know this issue is very similar to #3111, but I don't have CompactMachines nor CompactCrafting, it seems to happen because of [ChunkLoaders](https://www.curseforge.com/minecraft/mc-mods/chunk-loaders), and according to a comment by Darkere, maybe a bad order in chunk loading. But I can't get rid of these chunkloaders, as I have network receivers scattered around the overworld, and some in the End. External storage is not an option either, as I only have access to vanilla storage otherwise.

#### What you expected to happen:
My disks and storage blocks should stay in place and keep displaying their content.

#### Steps to reproduce:
1. Reload the world
2. All of the Refined Storage disks will vanish from their respective disk drives, and storage blocks will become empty.

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.16.5
- Forge: 36.2.0
- Refined Storage: 1.9.15

Does this issue occur on a server? no

#### If a (crash)log is relevant for this issue, link it here:

https://gist.github.com/Pyeroh/b77fe136bc17edc7ae371e024fde95aa

**Comment by Darkere** [comment]
There are plenty of alternative chunkloaders. FTBChunks is pretty popular.

That said I'll see if I can figure out what the other dev is doing wrong so I can report it.

**Comment by Pyeroh** [comment]
Indeed, I can rely on another chunkloader, but I'm playing on a modpack, so using these chunkloaders are intended as part of the progression. I could also rely only on external storage, using big chests added by another mod, but I would have to add that mod to the modpack.
All in all, I'd rather update mods than adding new ones 😕

**Comment by Darkere** [comment]
Duplicate of #3299
